libnvme/meson.build

```diff
@@ -46,7 +46,7 @@ if have_python_support
     configure_file(
         input:  '__init__.py', 
         output: '__init__.py',
-        copy: true,
+        configuration: conf,
```

Comment:
Right, the 0.49.0 is just for the dictionary type.

Reply:
But still it's an unrelated change and it's not even mentioned in commit message.

Reply:
I don't quite understand what the issue is. I just wanted to add libnvme's version to the python package so that one can do as follows:
```
import libnvme
libnvme.__version__ == "1.0"
True
```
If that causes a problem then let's just forget it.

Reply:
The problem is that I have to guess why you want to change the meson file. That's all. I have no problem with the change but I want to understand it. Is this needed?

Reply:
Yeah, that is also what I figured but I had to check the documentation.